cloudbuild/README.MD

```diff
@@ -0,0 +1,21 @@
+# Continuous Integration via Cloud Build
+
+[Cloud Build](https://cloud.google.com/cloud-build/) is configured to trigger on
+`/gcbrun` comment from maintainer in PR to the `master` branch.
+
+Cloud Build triggers a set of build steps, configured in a `cloudbuild.yaml`
+config file. These steps are:
+
+1.  Create a presubmit Docker container with initialization actions code and
+    necessary dependencies installed; this includes git, gcloud, fastunit and
+    parametrized.
+1.  Push a presubmit Docker container to
```

Comment:
s/a presubmit Docker container/the presubmit Docker container

Reply:
Done.